api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
+
+{Before using this template, read [the accompanying guide](api-reference-guide.md) to this template}.
+
+
+## Overview
+
+This reference describes the Application Program Interfaces (APIs) that help you {access | customize | program} {product}'s {features | functionality}.
+
+### Base URL
+
+The APIs referenced in this documentation use the following base URL:
+
+```
+https://api.example.com
+```
+
+
+### Authorization
+
+All requests to the API must be authenticated and authorized before they can access or manage resources on the {environment | platform}.
+
+The {product} API uses the `{auth_type}` authorization type.
+
+You can authenticate your requests by using {authentication type and method}.
+
+For example:
+
+```
+{EXAMPLE}
+```
+
+
+API requests without valid authentication will fail with a status of `{error_code}`.
+
+
+### Versioning
```

Comment:
I would changee this to:
### Version

And then just list the version number. Separate versions of APIs should have entirely separate documentation sets, in my opinion. You can even version your documentation to match the API versions.